src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -140,7 +140,7 @@ public override void VisitNamedType(INamedTypeSymbol symbol)
             if (!format.MiscellaneousOptions.IncludesOption(SymbolDisplayMiscellaneousOptions.ExpandNullable))
             {
                 //if we're expanding nullable, we just visit nullable types normally
-                if (IsNullableType(symbol) && !symbol.IsDefinition)
+                if (ITypeSymbolHelpers.IsNullableType(symbol) && !symbol.IsDefinition)
```

Comment:
:bulb: `symbol.IsNullableType()`

Reply:
> symbol.IsNullableType()

This will not work. The method is not member or an extension method.

Reply:
➡️ Yep, I saw later in the review that there was more than one `IsNullableType` helper method. I incorrectly assumed from the title of this pull request that the resulting code had just one.